image-layout.md

```diff
@@ -7,7 +7,7 @@ Given an image layout a tool can convert a given ref into a runnable OCI Image F
 The image layout has two top level directories:
 
 - "blobs" contains content-addressable blobs. A blob has no schema and should be considered opaque.
-- "refs" contains descriptors pointing to an image manifest list
+- "refs" contains descriptors pointing to an image manifest
```

Comment:
On Wed, Jun 15, 2016 at 08:22:22AM -0700, Vincent Batts wrote:
> > -- "refs" contains descriptors pointing to an image manifest list
> > +- "refs" contains descriptors pointing to an image manifest
> 
> refs are going to be a descriptor, though they ought to point to a
> manifest or manifest list

refs (being descriptors) *tell* you what they point to.  Folks
interested in having oci-image-tool (or other tool that just
understands the current version of this spec) be able to unpack their
ref should point them at
application/vnd.oci.image.manifest.list.v1+json or
application/vnd.oci.image.manifest.v1+json.  But there's no problem
with image-layout if they have a ref that points at a different media
type (e.g. maybe they have an extention, or are pointing at
application/vnd.oci.image.manifest.list.v2+json).  The image-layout
format (and contents of oci-layout) can still be the same.